hack/.vendor-helpers.sh

```diff
@@ -112,7 +112,7 @@ clean() {
 	local prune=( $(find vendor -depth -type d -not '(' "${findArgs[@]}" ')') )
 	unset IFS
 	for dir in "${prune[@]}"; do
-		find "$dir" -maxdepth 1 -not -type d -exec rm -v -f '{}' +
+		find "$dir" -maxdepth 1 -not -type d -not -name "LICENSE*" -exec rm -v -f '{}' +
```

Comment:
Some projects use "COPYING" instead of "LICENSE", but I don't know that
it's very common in Go projects.